lib/adapters/REST/endpoints/bulk-action.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { AxiosInstance } from 'contentful-sdk-core'
+import { GetBulkActionParams, GetSpaceEnvironmentParams } from '../../../common-types'
+import { BulkActionProps } from '../../../entities/bulk-action'
+import { RestEndpoint } from '../types'
+import * as raw from './raw'
+
+export const get: RestEndpoint<'BulkAction', 'get'> = (
```

Comment:
Should we define a return type shape for this?

Reply:
They get the return from the `plain/common-types.ts` file; but I added it here too